.github/workflows/ci.yml

```
@@ -18,8 +18,10 @@ jobs:
         run: uv python install 3.11
 
       - name: Install the dependencies
-        run: uv sync 
-
+        run: |
+          uv sync
+          
```

yamllint

[error] trailing spaces

(trailing-spaces)